PartnerMeeting.Rmd

```diff
@@ -0,0 +1,26 @@
+---
+title: "Partner meeting"
+author: "Vizlab, Data Science Branch, IIDD"
+output: html_document
+---
+
+### Interactive map
+This map shows the focal saline lakes in the western US and the associated data.
+This data includes the subbasins (HUC8), streams within these subbasins (stream order >=3), and stream gages within the subbasins.
+```{r, echo = FALSE, out.width = '100%'}
```

Comment:
you can add `warnings = FALSE` to the code chunk header here so that the package load messages don't appear in the export